src/test/java/uk/gov/register/functional/HomePageFunctionalTest.java

```diff
@@ -57,23 +58,4 @@ public void homepageHasCopyrightInFooterRenderedAsMarkdown() throws Throwable {
         Elements links = copyrightParagraph.select("a");
         assertThat(links.size(), greaterThan(0));
     }
-
-    @Test
-    public void homepageUsesRegisterIdForNameIfNameNotSpecified() {
-        Response response = register.getRequest(REGISTER_WITH_COPYRIGHT_FIELD, "/");
-        Document doc = Jsoup.parse(response.readEntity(String.class));
-
-        Elements registerNameHeading = doc.select("main .heading-large");
-        assertThat(registerNameHeading.first().text(), equalTo("Postcode register"));
-    }
-
-    @Test
-    public void homepageUsesRegisterNameIfExists() {
```

Comment:
These two tests are still relevant if we are correctly using `friendlyRegisterName` as above.

Reply:
Ignore this as per discussion